src/games/strategy/engine/ClientContext.java

```diff
@@ -43,6 +46,9 @@
   private final MapListingSource mapListingSource;
   private final ScrollSettings scrollSettings;
   private final FolderSettings folderSettings;
+  private final AiSettings aiSettings;
```

Comment:
ClientContext wires in the settings objects. This decouples the settings window and the settings value. One day soon we will replace ClientContext with dagger. For now, the 'proper' way to get a settings value is to get it via ClientContext, with constructor dependency injection. Grant it, we are in a half-way state, so this ideal is not quite reached, but that is the goal.